source/testnet/net/references/node-runner-service-configuration.rst

```diff
@@ -0,0 +1,281 @@
+.. include:: ../../variables.rst
+.. _node-runner-service-configuration:
+
+============================================
+Concordium node runner service configuration
```

Comment:
The title should probably be changed to emphasise that this is specific to Windows.